modules/47_regipol/module.gms

```diff
@@ -8,11 +8,26 @@
-*'               It overwrites all other carbon prices for selected regions (for example those implemented by 45_carbonprice or 46_carbonpriceRegi).
+*'               The `regiCarbonPrice` realization has two purposes. First, it allows to determine region specific year or budget targets for CO2 or GHG emissions.
+*'               Second, it comprises region-specific adjustments that are always active in this realization and policies that can be activated by specific switches (see bounds file).
+*'               On the emissions targets: Emissions targets can be activated via the switch cm_emiMktTarget and there are number of options that can be chosen to specify them
+*'               like the type of target (annual target or budget), the target year, the regions the target should be applied to,
+*'               the emissions metric (CO2, GHG, CO2 excl. bunkers/LULUCF etc.) and the emissions market (all, ETS, ESR). For details on the options please see the description of the switch.
+*'               When an emissions target is set via this switch, the carbon price in the target years and regions, to which it is applied, is adjusted over nash iterations in REMIND
+*'               until the desired emissions target is reached within a certain margin of tolerance defined by cm_emiMktTarget_tolerance. Note that also multiple emissions targets can be specified for
+*'               different years (e.g. the EU's 2030 and 2050 targets). The carbon price trajectory up to the target years are linear and the carbon price
+*'               is assumed to still increase linearly at a small rate after the last target year. Please check the parameter pm_emiMktTarget_dev_iter to see the emissions target convergence over iterations.
+*'               Note that these regional emissions targets of the module only overwrite the carbon price trajectory for the regions their are applied to, while the other regions keep the carbon price trajectories
+*'               that are adjusted in other parts of the model, in particular, from the global emissions target adjustment.
+*'               This means that these regional targets come on top of the global emissions target that REMIND aims to achieve.
+*'               On the regional bounds and adjustments: In the bounds file, there are a number of regionally hard-coded bounds to the model that aim to improve the representation of specific regions in REMIND.
+*'               They come on top of the bounds in the core of REMIND. The difference is that core defines bounds for all regions, while in this module region-specific adjustments are made which have not yet been generalized to all regions
+*'               and are not activated outside ot the realization "regiCarbonPrice".
+*'               These bounds either serve to align REMIND with historic and near-term data in specific regions or represent regional policies (like national coal phase-out plans) that can be activated via switches.
```

Comment:
just out of curiosity, is there a way we keep track of the status of these policies, i.e. link them? Sometimes they are rolled back, or delayed, for political reasons. Maybe a different archival structure is needed to curate these policies? Ideally, a newbie to REMIND can look at the code here and be directly linked to the policy document, or some explanatory text for it...

Reply:
sorry I just scrolled below: I like on line 133 there is a link to the policy... should there be one for the uk as well?